src/js/reducers/showBarReducer.jsx

```diff
@@ -0,0 +1,24 @@
+import {
+  ADD_INFO_BAR,
+  CLOSE_INFO_BAR,
+  HIDE_INFO_BAR,
+  SHOW_INFO_BAR,
+} from 'actions/types';
+import { InfoBar } from 'consts/infoBar';
+
+const initialState = Object.keys(InfoBar).reduce((acc, bar) => ({ ...acc, [bar]: false }), {});
+
+export default function showBarReducer(state = initialState, action) {
```

Comment:
We have to sort out the reducers and action types namings and states on both reducers